modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -43,8 +45,34 @@ m|INFORMATION
 m|PERFORMANCE
 |===
 
-.Cartesian product
-====
+======
+[.include-with-GQLSTATUS-code]
+======
+
+[cols="<1s,<4"]
+|===
+|GQLSTATUS code
+m|03N90
+|StatusDescription
+a|info: cartesian product.
+The disconnected patterns `%pat` build a cartesian product.
```

Comment:
I think we should use $ instead of % in front of the parameters, unless you have some certain reason for this way of writing? 

Also applies to the other StatusDescription fields in the PR